bundles/org.openhab.binding.pilight/src/main/resources/OH-INF/i18n/pilight_de.properties

```diff
@@ -19,6 +19,8 @@ thing-type.pilight.generic.label = Generisches pilight Gerät
 thing-type.pilight.generic.description = Gerät bei dem die Kanäle dynamisch hinzugefügt werden.
 
 # thing type config description
+thing-type.config.pilight.bridge.backgroundDiscovery.label = Hintergrund-Discovery
+thing-type.config.pilight.bridge.backgroundDiscovery.description = Sollen Pilight Geräte für diese Brigde automatisch erkannt werden.
```

Comment:
Please do not update German translations. This has to be done later in Crowdin.

Reply:
Done